motoman_driver/MotoPlus/MotionServer.c

```diff
@@ -797,7 +957,16 @@ BOOL Ros_MotionServer_StartTrajMode(Controller* controller)
 			goto updateStatus;			
 		}
 	}
-	
+
+	// have to initialize the prevPulsePos that will be used when interpolating the traj
+	for(grpNo = 0; grpNo < MP_GRP_NUM; ++grpNo)
+	{
+		if( controller->ctrlGroups[grpNo] != NULL /*&& Ros_CtrlGroup_IsRobot(controller->ctrlGroups[grpNo])*/)

```

Comment:
Should commented code be removed? Otherwise add comment why it should be retained.